bitrise.yml

```diff
@@ -60,13 +61,13 @@ workflows:
         inputs:
         - service_account_json_key_path: $BITRISEIO_JSON_KEY_URL
         - package_name: $PACKAGE_NAME
-        ## - app_path: $BITRISE_APK_PATH\n$BITRISE_AAB_PATH
+        - app_path: $BITRISE_SIGNED_AAB_PATH
```

Comment:
NOTE: in the above `sign-apk`, pls use the not-deprecated `android_app` input instead of `apk_path `